deployments/scripts/deploy-osmo-minimal.sh

```
@@ -358,9 +481,77 @@ verify_provider_config() {
+
+    local extra_args=()
+    if [[ "$AUTH_METHOD" == "workload-identity" ]]; then
+        extra_args+=(--auth-method workload-identity)
+        [[ -n "$WORKLOAD_IDENTITY_CLIENT_ID" ]] && \
+            extra_args+=(--workload-identity-client-id "$WORKLOAD_IDENTITY_CLIENT_ID")
+        [[ -n "$WORKLOAD_IDENTITY_ROLE_ARN" ]] && \
+            extra_args+=(--workload-identity-role-arn "$WORKLOAD_IDENTITY_ROLE_ARN")
+    fi
+    [[ "$NON_INTERACTIVE" == "true" ]] && extra_args+=(--non-interactive)
+
+    OSMO_NAMESPACE="${OSMO_NAMESPACE:-osmo-minimal}" \
```

Shellcheck (0.11.0)

[warning] This assignment is only seen by the forked process.

(SC2097)